### PR TITLE
Use default Django handlers for 400, 403, 404, and 500 errors

### PR DESCRIPTION
**Pull request by Jkrzy**
## Description

For #842, include a `400.html` template and fallback to default Django error handlers.

## Additional information

Current implementation for these handlers provides no functionality beyond what Django provides by default.  

[views.defaults.permission_denied](https://github.com/django/django/blob/190aa594477a62a43ca78c41d2d0af47dd6ccbba/django/views/defaults.py#L100) for example, renders the `403.html` template with the exception made available in the template context.

### CODE REVIEW

File: tock/tock/templates/400.html [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{% extends "base.html" %}
+
+{% block content %}
+<h2>Bad Request</h2>
```
Comment (tbaxter-18f) [inline tock/tock/templates/400.html]: Let's ping someone from content or writing lab and see if they have any suggestions on more user-friendly copy here.

File: tock/tock/urls.py [hunk 2]
```diff
@@ -23,10 +23,6 @@ def check_if_staff(user):
 import projects.urls
 import tock.views
 
-handler400 = 'tock.views.handler400'
-handler403 = 'tock.views.handler403'
-handler404 = 'tock.views.handler404'
-handler500 = 'tock.views.handler500'
```
Comment (tbaxter-18f) [inline tock/tock/urls.py]: I love the smell of red backgrounds in a pull request. Smells like... victory.

File: tock/tock/templates/400.html (hunk 1 above)
Comment (annepetersen) [reply tock/tock/templates/400.html]: @amandacostello can be of assistance. Looping in.

Comment (tbaxter-18f) [reply tock/tock/templates/400.html]: @amandacostello any update?

Comment (amandacostello) [reply tock/tock/templates/400.html]: Hi! Still getting my feet under me with GitHub. You're looking for copy related to just 400 error returns, or also 403, 404, and 500?

Comment (tbaxter-18f) [reply tock/tock/templates/400.html]: In particular, 400, but if there was general guidance for the others we'd be happy to incorporate that as well.

Comment (amandacostello) [reply tock/tock/templates/400.html]: I've got some writing for each, though what usually triggers the 400 on the client side? Also, as a new GitHub person, how/where should I put the new content? Or just put it in a comment here?

Comment (amandacostello) [reply tock/tock/templates/400.html]: ### Error: Bad Request (400)
We can't process your request due to something wrong on your end, like a message being too large or (other common example seen in Tock)

### Error: Access forbidden (403)
Your account doesn't have access. Make sure you're logged in, check your VPN connections, and that your account has the correct permissions enabled. 

### Error: File not found (404)
Looks like the file you're requesting isn't here anymore. Contact (name/email) or try (search)

### Error: Internal Server Error (500)
Something's gone wrong on our side. Contact (name/email) to let them know.

**Comment by tbaxter-18f** [comment]
@Jkrzy Do you think we'll be able to get the content updates in for the March 5 release?

**Comment by Jkrzy** [comment]
Thanks @amandacostello!! We're going to roll the content updates into a new issue and close this one, focused on the back-end implementation.

**Pull request merged**